cypress/integration/gov-create-opp.spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+
+describe('As a user authenticated via IDIR', function() {
+    beforeEach(function() {
+        // clean up db
+        cy.exec('dropdb -f digitalmarketplace')
+        // set up db
+        cy.exec('createdb digitalmarketplace')
+        cy.exec('npm run migrations:latest;')
+        cy.exec('psql digitalmarketplace -f ./cypress/fixtures/users.sql')
+        cy.visit('auth/createsession')
+        Cypress.Cookies.preserveOnce("sid")
+    })
+
+    it('creates and saves a draft of a new CWU opportunity', function() {
+
+        cy.visit("/opportunities/create")
+        cy.contains('Get Started').first().click()
+
+        // 1. Overview tab
+        cy.get('#cwu-opportunity-title').type('Cypress Opp')
+        cy.get('#cwu-opportunity-teaser').type('Teaser text')
+        cy.get('#cwu-opportunity-remote-ok-0').check({force:true}) // is force OK?
+        cy.get('#cwu-opportunity-remote-desc').type('Remote description text')
+        cy.get('#cwu-opportunity-location').clear().type('Vancouver')
+        cy.get('#cwu-opportunity-reward').type('5000')
+        cy.get('#cwu-opportunity-skills').click()
+        cy.contains('Agile').click({force: true})
```

Comment:
This is in a dropdown and what I've got here works, but it seemed sketchy?

Reply:
Ideally we'd wait for the click that opens the dropdown to finish opening, ensuring this click is reachable (so we don't have to force). Tech debt?

Reply:
https://github.com/button-inc/digital_marketplace/issues/118